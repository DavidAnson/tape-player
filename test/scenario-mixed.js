// @ts-check

"use strict";

const tape = require("tape");
require("../index.js");

tape("passing test", (test) => {
  test.plan(1);
  test.pass();
});

tape("failing test", (test) => {
  test.plan(1);
  test.fail("oops");
});

tape("passing test", (test) => {
  test.plan(1);
  test.pass();
});
