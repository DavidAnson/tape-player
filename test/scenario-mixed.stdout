passing test
failing test
  /home/pi/tape-player/test/scenario-mixed.js:15:8
  Message:  oops
  Operator: fail
passing test

Tests:    3
Asserts:  3
Failures: 1
